### PR TITLE
Table prefix

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

**Comment by fbertsch** [comment]
This will make the tablenames more readable. We're going to have datasets named e.g. `firefox_android_release_external` and `firefox_ios_release_external`, so we want to prepend a name to the table to indicate it is from leanplum. We'll run this with `--table-prefix leanplum`.

Additionally, this removes the `output` from the table name, so it will be something like `firefox_android_release_external.leanplum_sessions`.

**Comment by fbertsch** [comment]
Also added optional versions, so tables will end up like this: `firefox_android_release_external.leanplum_sessions_v1`.

File: README.md [hunk 1]
```diff
@@ -32,7 +34,8 @@ make run COMMAND="leanplum-data-export export-leanplum \
   --date 20190101 \
   --bucket gcs-leanplum-export \
   --prefix dev \
-  --dataset leanplum"
+  --table-prefix leanplum \
+  --bq-dataset leanplum"
```
Comment (akkomar) [inline README.md]: nit: different ordering than above.

**Review by akkomar — DISMISSED** [review]
LGTM

**Pull request merged**